book/06-github/sections/2-contributing.asc

```diff
@@ -71,7 +71,8 @@ $ cd blink
 $ git checkout -b slow-blink <2>
 Switched to a new branch 'slow-blink'
 
-$ sed -i '' 's/1000/3000/' blink.ino <3>
+$ sed -i '' 's/1000/3000/' blink.ino <3> (MacOSX)
+$ sed -i 's/1000/3000/' blink.ino <3> (Linux)
```

Comment:
sounds good to me, changed accordingly in 88d99d440d6c1d0d9aa74d3609d3227ddb400b5e